Makefile

```diff
@@ -9,3 +9,9 @@ test:
 
 run:
 	go run ./cmd/...
+
+update:
+	go list -m -u all \
+	| awk -F" " '{ if ($$3 != "") print $$1 " " $$3; }' \
```

Comment:
it's not working on my machine

` go list -m -u all | awk -F" " '{ if ($$3 != "") print $$1 " " $$3; }'

awk: illegal field $(), name "3"
 input record number 1, file
 source line number 1`

Reply:
Mhhhm. That surprises me a bit. Can you run awk -h and posted it's output?

Reply:
awk

usage: awk [-F fs] [-v var=value] [-f progfile | 'prog'] [file ...]

awk -h

awk: unknown option -h ignored

awk: no program given